lms/djangoapps/instructor_task/api.py

```diff
@@ -576,6 +577,6 @@ def export_grades_to_rgb(request, course_key, assignment_name, email):
         "assignment_name": assignment_name,
         "email_id": email
     }
-    task_key = ""
+    task_key = uuid.uuid4()
```

Comment:
I don't think that we need to use a UUID for the task key, as it would still be useful to prevent an arbitrary number of jobs that are trying to process the same assignment. I think that if we make the assignment_name the task key, or a hash of the assignment name and the course_id, then that would allow for multiple jobs to be submitted for a single course, across the different assignments, while still preserving the original intent of preventing duplicate tasks from being submitted erroneously.

@pdpinch please correct me if my understanding of the intent is misinformed.

Reply:
sounds good to me @pdpinch any thoughts?

Reply:
Sorry I missed this. What Tobias describes is exactly what I had in mine.